src/components/EditProfileModal/EditProfileModal.tsx

```diff
@@ -56,9 +56,13 @@ export const EditProfileModal = ({
   return (
     <EditProfileForm.FormController
       source={myProfile}
-      submit={params =>
-        updateMyProfile(params).then(onClose).catch(console.warn)
-      }
+      submit={params => {
+        const fixedParams: any = { ...params };
+        const skills = params.skills;
+        delete fixedParams.skills;
+        fixedParams.skills = JSON.stringify(skills);
+        updateMyProfile(fixedParams).then(onClose).catch(console.warn);
+      }}
```

Comment:
@CryptoGraffe I hope this refactor isn't too over the top:
```suggestion
            submit={async params => {
        const fixedParams: Omit<typeof params, 'skills'> & { skills: string } =
          { ...params, skills: JSON.stringify(params.skills) };
        try {
          await updateMyProfile(fixedParams);
          onClose();
        } catch (e: unknown) {
          console.warn(e);
        }
      }}

```

Reply:
ok I applied this, plus a change that allows nulling of website via empty string